assets/js/vue/pages/Dataland.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+    <div class="container">
+        <div class="row">
+            <main class="col-lg-9 overflow-auto">
+                <div class="pt-3">
+                    <h2> {{ dataset.title }} </h2>
+                </div>
+                <div class="d-flex justify-content-between py-2">
+                    <div v-if="Object.keys(dataset.datasetSubmission).length > 0">
+                        <div v-if="dataset.datasetSubmission.authors">
+                            <strong>Authors:</strong> {{ dataset.datasetSubmission.authors }}
+                        </div>
+                        <div v-if="dataset.acceptedDate">
+                            <strong>Published on </strong> {{ getFormattedDate(dataset.acceptedDate.date, 'MMMM Do YYYY') }}
+                        </div>
+                        <div v-if="dataset.availabilityStatus !== 7 && dataset.datasetSubmission.distributionFormatName">
+                            <strong>File Format:</strong> {{ dataset.datasetSubmission.distributionFormatName }}
+                        </div>
+                    </div>
+                    <div>
+                        <div v-if="dataset.availabilityStatus !== 0 && dataset.doi.doi">
+                            <strong>DOI:</strong> {{ dataset.doi.doi }}
+                        </div>
+                        <div>
+                            <strong>UDI:</strong> {{ dataset.udi }}
+                        </div>
+                        <div v-if="dataset.availabilityStatus !== 7 && dataset.datasetSubmission.fileset">
```

Comment:
I don't like having so much logic and these pages. Also can we translate the numbers to constants?
Even with Twig we would have logic. Maybe we should retool the API, and not give values for DIVs we don't want to show, then just check if values exist, then show or not show div.

Thoughts?